internal/i18n/locales/en/default.po

```diff
@@ -0,0 +1,16 @@
+msgid ""
+msgstr ""
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid "create-verification-code"
```

Comment:
I'd like to come up with a better grouping for these. I thought about `header.create-verification-code` or something to make it clearer.